data/hypervisor.args.in

```diff
@@ -2,7 +2,7 @@
 -name
 @NAME@
 -machine
-pc-lite,accel=kvm,kernel_irqchip,nvdimm
+pc,accel=kvm,kernel_irqchip,nvdimm
```

Comment:
Agreed. Anthony was involved in the debug which resulted in the above conclusions.

This should be a temporary issue until fixes are made, but we don't have any ETA at this point. I'd like to get these changes on and then add parameters for vm memory, and then see where we are for machine type, vmlinuz changes.